arch/simddetect.cpp

```diff
@@ -26,7 +26,7 @@
 #endif // x86 target
 
 #if defined(X86_BUILD)
-# if defined(__linux__) || defined(__MINGW32__) || defined(__APPLE__)
+# if defined(__GNUC__) || defined(__clang__) || defined(__MINGW32__) || defined(__MINGW64__) 
```

Comment:
We don't use `__MINGW64__` elsewhere in the codebase, we only use `__MINGW32__`.